checkpointers/empty/checkpointer.go

```diff
@@ -0,0 +1,34 @@
+package emptycheckpointer
+
+import (
+	k "github.com/remind101/kinesumer/interface"
```

Comment:
(https://github.com/remind101/kinesumer/blob/multi-consumer/interface/kinesis.go was copied from an older version of that)